rasa/core/training/story_conflict.py

```diff
@@ -0,0 +1,221 @@
+from collections import defaultdict, namedtuple
+from typing import List, Optional, Dict, Text
+
+from rasa.core.actions.action import ACTION_LISTEN_NAME
+from rasa.core.domain import PREV_PREFIX, Domain
+from rasa.core.events import ActionExecuted, Event
+from rasa.core.featurizers import MaxHistoryTrackerFeaturizer
+from rasa.nlu.constants import MESSAGE_INTENT_ATTRIBUTE
+from rasa.core.training.generator import TrackerWithCachedStates
+
+
+class StoryConflict:
+    def __init__(self, sliced_states: List[Optional[Dict[Text, float]]],) -> None:
+        self.sliced_states = sliced_states
+        self._conflicting_actions = defaultdict(
+            list
+        )  # {"action": ["story_1", ...], ...}
+        self.correct_response = None
```

Comment:
can you please add an attribute type?
```suggestion
        self.correct_response: Optional[??] = None
```

Reply:
I deleted the attribute, since it's no longer needed.